lib/ui/text.dart

```diff
@@ -2123,7 +2123,20 @@ class ParagraphBuilder extends NativeFieldWrapperClass2 {
 Future<void> loadFontFromList(Uint8List list, {String fontFamily}) {
   return _futurize(
     (_Callback<void> callback) => _loadFontFromList(list, callback, fontFamily)
-  );
+  ).then(_sendFontChangeMessage);
+}
+
+final ByteData _fontChangeMessage = utf8.encoder.convert(
+  json.encode(<String, dynamic>{'type': 'fontsChange'})
+).buffer.asByteData();
+
+FutureOr<void> _sendFontChangeMessage(void result) async {
```

Comment:
nit: I think it would be clearer if this function didn't have an unused parameter named `result` and instead we wrote `.then((_) => _sendFontChangeMessage())`

But it's not a hard preference